pkg/api/api.go

```diff
@@ -59,6 +59,10 @@ func New(s storage.Storage, options Options) *API {
 }
 
 func response(c echo.Context, data interface{}, dataType string, p ...storage.Pagination) error {
+	if data == nil {
+		return c.NoContent(http.StatusOK)
```

Comment:
Can we change the response to be a 204-No Content status?

ref: https://restfulapi.net/http-methods/#:~:text=A%20successful%20response%20of%20DELETE,DELETE%20operations%20are%20idempotent.

Reply:
Sure! Will fix it.